src/lib/sentry.ts

```diff
@@ -1,8 +1,46 @@
+/**
+ * Importing these separately from `platform/detection` and `lib/app-info` to
+ * avoid future conflicts and/or circular deps
+ */
+
+import {Platform} from 'react-native'
+import app from 'react-native-version-number'
```

Comment:
Let's use Expo Application instead to make sure we are getting the correct value from app.config.js [docs.expo.dev/versions/latest/sdk/application/#applicationnativebuildversion](https://docs.expo.dev/versions/latest/sdk/application/#applicationnativebuildversion)